modules/runtime/pages/process-manager-application.adoc

```diff
@@ -37,14 +37,51 @@ You don't need to export and modify the orphan pages.
 [WARNING]
 ====
 
-Currently, the Monitoring page is made using a different technology than UI Designer, and thus it cannot be edited.
+Currently, the Monitoring page is made using a different technology than UI Designer, and thus it needs to be updated differently.
 ====
 
 Perform the following modifications to the 4 menu pages:
 
 1. Though not necessary, it is highly recommended to re-save your pages with a different name. This will prevent these pages from being over-written if you need to re-import the default pages.
 2. Add an api call to get current user session. Create a new "External API" variable called `session` with the value `../API/system/session/unusedId`
-3. Since a process manager only supervises a subpart of all the processes/cases/tasks, some API calls must be updated to only retrieve the expected processes/cases/tasks. To do so, the filter `supervisor_id` has to be used with the current user ID: Add `&f=supervisor_id={{session.user_id}}` to the variables of the "External API" type (except for the `session` variable that we created).
+3. Since a process manager only supervises a subpart of all the processes/cases/tasks, some API calls must be updated to only retrieve the expected processes/cases/tasks. To do so, the filter `supervisor_id` has to be used with the current user ID: Add `&f=supervisor_id={{session.user_id}}` to the query parameters inside the "Javascript expression" type variables that ended with "Ctrl" on its name. For example; on the process list page, inside "enabledProcessCtrl" and "disabledProcessCtrl", add `"&f=supervisor_id=" + $data.session.user_id` filter string to the "queryParameters" variable.
```

Comment:
```suggestion
3. Since a process manager only supervises a subpart of all the processes/cases/tasks, some API calls must be updated to only retrieve the expected processes/cases/tasks. The filter `supervisor_id` has to be used with the current user ID in order to get the subpart of all elements that he manages.
To do so, modify all the query parameters functions inside the "Javascript expression" type variables who's names end with "Ctrl". Add ` + "&f=supervisor_id=" + $data.session.user_id` to the result of all such functions.
For example: on the process list page, inside "enabledProcessCtrl" and "disabledProcessCtrl", add `"&f=supervisor_id=" + $data.session.user_id` string to the "queryParameters" variable.
```